### DISCUSSION TITLE
Custom User Agent Line in Bromite WebView

### DISCUSSION THREAD

**Discussion by sergiok707** [bromite/bromite#1659] [General]
I watched
available Bromite WebView activities. And I found a menu with debug settings there. But didn't find custom User Agent Line. It is in Bromite Browser. But I need a Bromite WebView. With this setting. Can you add this please

**Comment by sergiok707** [comment]
@csagan5 @uazo Can you help me please?
This is very important for my project
If everything works out, I will definitely donate as respect and gratitude.
Thanks

**Reply by uazo** [reply]
> But didn't find custom User Agent Line

correct, there isn't. 

> Can you add this please

the webview already has it
ref https://developer.android.com/reference/android/webkit/WebSettings#setUserAgentString(java.lang.String)

> If everything works out, I will definitely donate as respect and gratitude.

no development for money here

**Reply by sergiok707** [reply]
Thanks for the quick response!

I tried it really works but only within my personal application, mini browser. Thank you

But how to change the UserAgent in the BromiteWebView itself, so that any application working through the WebView would use the UserAgent string set in the Bromite WebView itself? I see a line like this in Bromite WebView 74.0.3729.80

Can you build an apk in which I can change this preset line to any other?

**Reply by csagan5** [reply]
> But how to change the UserAgent in the BromiteWebView itself, so that any application working through the WebView would use the UserAgent string set in the Bromite WebView itself? I see a line like this in Bromite WebView 74.0.3729.80

That is not possible and has never been possible.

> Can you build an apk in which I can change this preset line to any other?

You can contact is company: https://www.igalia.com/project/chromium (unrelated to Bromite, I found it by searching online)

> Chromium is a well-known open source web browser project started by Google. Igalia engineers have extensive experience on the Chromium project and have several Owners and Committers within our team. Our position and contacts in the community place Igalia as the most qualified consultancy to help you with your Chromium needs.

**Reply by sergiok707** [reply]
Okay, thanks
